.github/workflows/push-to-docker.yml

```diff
@@ -0,0 +1,31 @@
+name: Push to Docker
+
+on: [workflow_dispatch]
+
+jobs:
+    docker-push:
+        runs-on: ubuntu-latest
+        env:
+            DOCKER_BUILDKIT: 1
+
+        steps:
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v1
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v1
+
+            - name: Login to DockerHub
+              uses: docker/login-action@v1
+              with:
+                  username: ${{ secrets.DOCKERHUB_USERNAME }}
+                  password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+            - name: Docker Build and Push
+              uses: docker/build-push-action@v2
+              with:
+                  context: .
+                  file: ./Dockerfile
+                  platforms: linux/amd64,linux/arm64,linux/386
+                  push: true
+                  tags: polymathian/sharpei:latest
```

Comment:
not sure about this tag @gricey432

Reply:
Have a look at the existing github action to see how we get versions for releases
https://github.com/Polymathian/sharpei/blob/master/.github/workflows/pythonpublish.yml

Reply:
Tags should now include the version number